HW06.AssemblyOne/Motorcycle.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace HW06.AssemblyOne
+{
+    public class Motorcycle
+    {
+        private string vinNumber;
```

Comment:
for private fields, we use the following name convention
`vinNumber` should be `_vinNumber`